python/src/trezorlib/device.py

```diff
@@ -233,7 +233,6 @@ def recover(
 @session
 def reset(
     client: "TrezorClient",
-    display_random: bool = False,
```

Comment:
this is API breaking, please keep the argument and instead add a warning if it is set to `True`

Reply:
[2df710a](https://github.com/trezor/trezor-firmware/pull/4119/commits/2df710af7743a2972e8136f0004f56d1d3054275)